hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1914,7 +1916,7 @@ public AbfsClient getClient() {
   }
 
   @VisibleForTesting
-  void setClient(AbfsClient client) {
+  public void setClient(AbfsClient client) {
```

Comment:
lets not public it.
because fs.getAbfsStore is also public.
anyone can change the client.

Better to add new tests in org.apach.hadoop.fs.azurebfs